docs/RemoteDesktopToVM.md

```diff
@@ -34,3 +28,36 @@ Install the [Microsoft Remote Desktop](https://apps.apple.com/us/app/microsoft-r
 ### Linux
 
 Install an appropriate remote desktop client. For Ubuntu, Remmina Remote Desktop Client should already be installed.
+## Connect to the VM
+
+Go to [desktops.gordon.edu](https://desktops.gordon.edu/) and sign-in with your Gordon account. Select either CPS Server 1 or CPS Server 2 to download the remote desktop profile to connect to that server. The servers are the same, but they have separate resources, so choose the one that fewer people are currently using.
+
+### Windows or MacOS
+
+Double click the downloaded link to open the connection to the remote desktop.  Continue below with [Authentication].(#authentication)
+
+### Linux/Remmina
+
+If using Remmina on Linux, use the following steps to configure the RDP connection, otherwise skip to [Other Systems] (#other-systems).
+
+1. Click on the "Show Applications" button at the bottom left of the screen and type "remmina" in the search bar.  Right-click on the Remmina icon and select "Add to Favorites" then left-click to open the app.
+
+1. Open the file browser and navigate the Downloads folder drag the `cpub-CPS_Server_1-CPS_Server_1-CmsRdsh.rdp` file (or the corresponding "2" file if you chose that machine) to the Remmina app window.  This will create an entry named `RD-BROKER.GORDON.EDU`.
+
+1. Right-click on `RD-BROKER.GORDON.EDU` entry and select "Edit"  If desired, you can change the name for this connection.
+
+1. Under the "Basic" tab enter your username, password (optional), and domain (both `GORDON` and `gordon.edu` work).  For Resolution, select "Use client resolution"
+
+1. Select the "Advanced" tab.  Feel free to change the quality of the connection then scroll down to the very bottom to find multiple checkboxes and make the following changes:
+   - Deselect "Share Printers"
+   - Select "Server detection using Remote Desktop Gateway"
+   - Select "Ignore certificate"

```

Comment:
```suggestion
   - Select "Ignore certificate"
   - Select "Use base credentials for gateway too"
```
I believe I suggested this to you originally in the ticket. This should make it so that you don't have to type your credentials twice. Did you have issues with that?

Reply:
I did see your note and used it when I configured Remmina in my laptop running Ubuntu 22.04.  That option is not available in the version of Remmina running in Ubuntu 20.04, which is the version installed on our workstations.   I'll add it to the instructions with a note saying "you may not have this option".

Reply:
Ah okay. That makes sense. I'm surprised as it has been a windows RDP feature for a while! Thanks for adding the note about it.